react/modules/sample/src/pages/employee/SampleComponents.js

```
@@ -1213,6 +1216,158 @@ const SampleComponents = () => {
+            }} // on click of close
+            footerChildren={[
+              <Button
+                type={"button"}
+                size={"large"}
+                variation={"primary"}
+                label={"Close"}
+                onClick={() => {
+                  setShowPopup(false);
+                }}
+              />,
+            ]}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)